server/helpers/token.js

```diff
@@ -1,8 +1,7 @@
 import jwt from 'jsonwebtoken';
-import tokenKeys from '../config/keys';
 
 const Token = (payLoad) => {
-  const Newtoken = jwt.sign({ payLoad }, tokenKeys, {
+  const Newtoken = jwt.sign({ payLoad }, process.env.tokenKeys, {
```

Comment:
It's best practice to name your variables in the dotenv file in a capital case format and use an underscore to separate words in this instance `TOKEN_KEY`.

Reply:
Done